styles/itsy.css

```diff
@@ -84,3 +84,49 @@ header {
   margin-bottom: 0px;
   line-height: 45px;
 }
+
+.moments {
+  padding: 20px;
+}
+
+.moments-title {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-size: 50px;
+  color: #808080;
+  margin: 60px;
+}
+
+.moments-img {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  flex-grow: 1;
+}
+
+.box-item {
+  display: flex;
+  flex-direction: column;
+  width: 32%;
+  margin-bottom: 20px;
+}
+
+.box-item__img {
+  border-radius: 50%;
+  height: 80%;
+  margin-bottom: 30px;
+}
+
+.box-item__title, .box-item__subtitle {
+  text-transform: uppercase;
+  text-align: center;
+  font-size: 30px;
+  font-weight: 700;
+  margin: 0px;
```

Comment:
Please add this selector to remove default margins/paddings:
`* {
margin: 0;
padding: 0;
}`